packages/ckeditor5-bookmark/src/bookmarkui.ts

```diff
@@ -54,6 +249,8 @@ export default class BookmarkUI extends Plugin {
 		const editor = this.editor;
 		const locale = editor.locale;
 		const view = new ButtonClass( locale ) as InstanceType<T>;
+		const insertCommand = editor.commands.get( 'insertBookmark' )!;
+		const updateCommand = editor.commands.get( 'updateBookmark' )!;
```

Comment:
```suggestion
		const insertCommand: InsertBookmarkCommand = editor.commands.get( 'insertBookmark' )!;
		const updateCommand: UpdateBookmarkCommand = editor.commands.get( 'updateBookmark' )!;
```

Reply:
There is no need in assigning the type because in typescript there is cool feature called "inference", its already knows the type.
![image](https://github.com/user-attachments/assets/613fbcc4-475b-4091-a974-b09ee5d8954a)